bloch_sphere/animate_bloch.py

```diff
@@ -142,6 +143,22 @@ def apply_gate_list(self, gates, final_wait=True):
                     print(f'Error: Custom gate arguments contain invalid '
                           f'floats {gate}.')
                 continue
+
+            if re.match('r[x,y,z][,;]', gate[:3].lower()):
+                try:
+                    r_pi = float(gate[3:])
+                except ValueError:
+                    print('Error: Rx/Ry/Rz gate should have style like '
+                          'Rx;{float} or Rx,{float}.')
+                gate_name = 'R' + gate[1].lower()  # Display Rx instead of rx
+                x = int(gate_name[1] == 'x')
+                y = int(gate_name[1] == 'y')
+                z = int(gate_name[1] == 'z') 
+                formated_r_pi = f"{r_pi:.3f}".rstrip('0').rstrip('.')
```

Comment:
Forgot one thing.  Add π after the number to match convention.

Reply:
Ah, Good point. Added